src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindCollectionsOperation.java

```diff
@@ -109,14 +112,35 @@ public CommandResult get() {
                               new CreateCollectionCommand.Options.VectorizeConfig(
                                   collectionProperty.vectorizeServiceName(), vectorizeOptions);
                         }
-                        CreateCollectionCommand.Options.VectorSearchConfig vectorSearchConfig =
+
+                        vectorSearchConfig =
                             new CreateCollectionCommand.Options.VectorSearchConfig(
                                 collectionProperty.vectorSize(),
                                 collectionProperty.similarityFunction().name().toLowerCase());
+                      }
+
+                      if (collectionProperty.indexingConfig() != null) {
+                        indexingConfig =
+                            new CreateCollectionCommand.Options.IndexingConfig(
+                                collectionProperty.indexingConfig().allowed() != null
+                                        && !collectionProperty.indexingConfig().allowed().isEmpty()
```

Comment:
If using `NON_EMPTY` inclusion earlier, could avoid checks here I think? (so `null` and empty Collection would just not be serialized.

Reply:
The IndexingConfig object in the cache has the allow and deny set initialized to empty, I did it so we don't have to do null check when trying to do contains check. Here I need to do not empty check so indexing options don't show up in findCollections as below
```
"indexing": {
  "allow" : [],
  "deny" : []
}
```

Reply:
Ok yes I realized it after reading later code. Yes, it simplifies later check so everything is fine as-is.